client/my-sites/hosting/main.js

```diff
@@ -143,27 +147,28 @@ class Hosting extends Component {
 				);
 			}
 
-			const failureNotice = FAILURE === transferState && (
-				<Notice
-					status="is-error"
-					showDismiss={ false }
-					text={ translate( 'There was an error activating hosting features.' ) }
-					icon="bug"
-				/>
-			);
+			if ( isBasicHostingDisabled && ! isTransferring ) {
+				const failureNotice = FAILURE === transferState && (
+					<Notice
+						status="is-error"
+						showDismiss={ false }
+						text={ translate( 'There was an error activating hosting features.' ) }
+						icon="bug"
+					/>
+				);
+
+				// Don't imply additional access if site doesn't have the SFTP feature.
+				const noticeText =
+					isBasicHostingDisabled &&
+					( englishLocales.includes( locale ) ||
+						i18n.hasTranslation( 'Please activate your hosting access.' ) )
+						? translate( 'Please activate your hosting access.' )
+						: translate( 'Please activate the hosting access to begin using these features.' );
```

Comment:
IMO the extra case—taking a WooExpress site Atomic—doesn't warrant its own copy. 1) it doesn't seem that likely 2) "these features" doesn't feel like it implies the SFTP to me at least, it's more implies the "advanced" features mentioned above in the sub title.

![woo express that is simple](https://github.com/Automattic/wp-calypso/assets/1500769/23cc8c3e-9011-49d2-a127-e6f77a6d356e)

Reply:
Agreed.

Reply:
And the one case where it would occur is for stuck transfers, which we have started tripping over. So this is more likely to be something seen by HEs, but they still need a path for re-triggering a transfer.